systems/daint-gpu.sh

```diff
@@ -9,13 +9,14 @@ ns_sysname="daint-gpu"
 module load daint-gpu
 module load CMake
 
-module load cudatoolkit/9.2.148_3.19-6.0.7.1_2.1__g3d9acc8
```

Comment:
Why  remove cudatoolkit here?

Reply:
Cut and paste errors here; will rectify.